src/main/resources/openapi.yaml

```
@@ -785,35 +898,34 @@ components:
           type: string
       xml:
         name: '##default'
-      type: object
   requestBodies:
     Pet:
+      description: Pet object that needs to be added to the store
       content:
         application/json:
           schema:
             $ref: '#/components/schemas/Pet'
         application/xml:
           schema:
             $ref: '#/components/schemas/Pet'
-      description: Pet object that needs to be added to the store
     UserArray:
+      description: List of user object
       content:
         application/json:
           schema:
             type: array
             items:
               $ref: '#/components/schemas/User'
-      description: List of user object
   securitySchemes:
     petstore_auth:
       type: oauth2
       flows:
         implicit:
-          authorizationUrl: 'https://petstore.swagger.io/oauth/authorize'
+          authorizationUrl: https://petstore3.swagger.io/oauth/authorize
           scopes:
-            'write:pets': modify pets in your account
-            'read:pets': read your pets
+            write:pets: modify pets in your account
```

Though it's valid YAML, using quotes here eliminates visual ambiguity and allows compatibility with wider range of YAML parsers.

```suggestion
            "write:pets": modify pets in your account
            "read:pets": read your pets
```

---

I would agree